ahorcado.rb

```diff
@@ -1,62 +1,44 @@
-#TODO: word_spaces should depend from current_word
-# Array equivalent to the characters of the word 
-# [“”, “”, “”, “”, ...]
-# []
-# Array.new
-
-#current_word = "love"
-#word_spaces = Array.new(current_word.length, "")
-
-@current_word = "friends"
-@word_spaces = Array.new(@current_word.length, "")
-@current_characters = @current_word.split("")
-
-def display_game
-  @word_spaces.each do |characters|
-    print "| #{characters} | "
+class Ahorcado
```

Comment:
I'm fine with using Spanish names for titles and stuff. However, have you thought about looking at the English name of this game?

Reply:
Yes, I will update that 💃